checkov/secrets/utils.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+import os
+import re
+from collections.abc import Iterable
+
+from checkov.common.runners.base_runner import ignored_directories, safe_remove
+from checkov.common.util.consts import DEFAULT_EXTERNAL_MODULES_DIR
+
+EXCLUDED_PATHS = [*ignored_directories, DEFAULT_EXTERNAL_MODULES_DIR, ".idea", ".git", "venv"]
+
+
+def filter_excluded_paths(
+    root_dir: str,
+    names: list[str] | list[os.DirEntry[str]],
+    excluded_paths: Iterable[str] | None,
+) -> None:
+    """Special build of checkov.common.runners.base_runner.filter_ignored_paths for Secrets scanning"""
+
+    # old logic
```

Comment:
is it planned to keep the old logic ? or is it just temp ?

Reply:
it will stay, because this for the case a user uses the `--skip-path` flag. Should I maybe reword it? I commented it with old, because it is from the old function 😄